projects/aca-playwright-shared/src/utils/utils.ts

```diff
@@ -98,4 +98,18 @@ export class Utils {
     const fileInput = await personalFilesPage.page.$('#app-upload-file-version');
     await fileInput.setInputFiles(path.join(__dirname, `../resources/test-files/${fileFromOS}.docx`));
   }
+
+  static async reloadPageIfRowNotVisible(
+    pageContext: PersonalFilesPage | MyLibrariesPage | FavoritesLibrariesPage | SearchPage,
+    nodeName: string,
+    errorMessage = 'Error '
+  ): Promise<void> {
+    try {
+      if(!await pageContext.dataTable.getRowByName(nodeName).isVisible()){
+        await pageContext.page.reload({waitUntil:"load"});
```

Comment:
please format it a bit 
{ waitUntil: 'load' }

(extra spaces + ' instead of ")